src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
```

Comment:
You're using the `body` tag for styling, which is against the checklist. You should use a class instead. As a general rule, avoid using tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.